charts/ceph-csi-rbd/templates/nodeplugin-daemonset.yaml

```
@@ -86,6 +86,10 @@ spec:
 {{- end }}
 {{- if .Values.nodeplugin.profiling.enabled }}
             - "--enableprofiling={{ .Values.nodeplugin.profiling.enabled }}"
+{{- end }}
+{{- if .Values.readAffinity.enabled }}
+            - "--enable-read-affinity=true"
```

This will be problematic if we choose to enable it by default later for some reason and the user should be able to disable it

---

@Madhu-1,  I didn't get you on this. Can you explain bit more?

---

```suggestion
            - "--enable-read-affinity={{  .Values.readAffinity.enabled }}"
 {{- if .Values.readAffinity.enabled }}
            - "--crush-location-labels={{ .Values.readAffinity.crushLocationLabels | join "," }}"
{{- end }}
```

The above change will allow users to disable the ReadAffinity if its get enabled by default.

---

Okay, I got it
Thanks!